packages/rpc-methods/src/permitted/getSnaps.ts

```diff
@@ -28,12 +28,17 @@ export type GetSnapsHooks = {
 };
 
 /**
- * @param _req
- * @param res
- * @param _next
- * @param end
- * @param options0
- * @param options0.getSnaps
+ * The `wallet_getSnaps` method implementation.
+ * Fetches available snaps for the requesting origin and adds them to the JSON-RPC response.
+ *
+ * @param _req - The JSON-RPC request object. Not used by this function.
+ * @param res - The JSON-RPC response object.
+ * @param _next - The `json-rpc-engine` "next" callback. Not used by this
+ * function.
+ * @param end - The `json-rpc-engine` "end" callback.
+ * @param hooks - The RPC method hooks.
+ * @param hooks.getSnaps - A function that returns the snaps available for the requesting origin.
+ * @returns Nothing.
```

Comment:
This shouldn't be necessary?

Reply:
It seems that it is, the linter fails if its not there.

Reply:
Maybe change it to "An empty promise" or something similar then?